crates/holochain/src/conductor/conductor.rs

```diff
@@ -1503,13 +1504,17 @@ mod app_impls {
 mod cell_impls {
     use super::*;
     impl Conductor {
-        pub(crate) async fn cell_by_id(&self, cell_id: &CellId) -> ConductorResult<Arc<Cell>> {
+        pub(crate) async fn cell_by_id(&self, cell_id: &CellId, require_network_ready: bool) -> ConductorResult<Arc<Cell>> {
```

Comment:
I don't think this should always be true, because you might actually want to be able to find the cell (e.g. dump its state) before it has joined the network. So this is a choice the caller has to make